Watchman.Discord/Areas/Users/Controllers/UsersController.cs

```diff
@@ -59,6 +59,7 @@ public void RemoveRole(DiscordRequest request, Contexts contexts)
             _rolesService.DeleteRoleFromUser(safeRoles, messagesService, contexts, commandRole);
         }
 
+        [DiscordCommand("list role")]
```

Comment:
jak już to robisz to przepisz na sposób z IRoleCommand
powinna być komenda o nazwie RolesCommand

Reply:
przy okazji zapoznasz sie z tym jak to działa